layers/meta-resin-qemu/recipes-bsp/grub/files/grub.cfg

```diff
@@ -2,5 +2,5 @@ default=resin
 timeout=3
 
 menuentry 'resin'{
-linux /bzImage root=/dev/hda2 rootwait
+linux /bzImage root=/dev/disk/by-label/resin-rootA rootwait
```

Comment:
@floion but I did add Update grub.cfg - Search QEMU root device by label instead of hda/sda/vda paths  - which indicates the same right?